tests/Fixtures/TestBundle/Entity/CompositeRelation.php

```diff
@@ -50,7 +50,7 @@ class CompositeRelation
      */
     public function getId()
     {
-        return sprintf('%s-%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
+        return sprintf('%s;%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
```

Comment:
My mistake, I think this is some leftovers from my tests with api-platform@1.x ref https://github.com/api-platform/core/issues/282#issuecomment-142578508